src/test/java/com/rultor/agents/github/ClosePullRequestTest.java

```diff
@@ -0,0 +1,134 @@
+ * copyright notice, this list of conditions and the following
+ * disclaimer in the documentation and/or other materials provided
+ * with the distribution. 3) Neither the name of the rultor.com nor
+ * the names of its contributors may be used to endorse or promote
+ * products derived from this software without specific prior written
+ * permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT
+ * NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+ * FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL
+ * THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+ * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+ * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+ * OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.rultor.agents.github;
+
+import com.jcabi.github.Comment;
+import com.jcabi.github.Issue;
+import com.jcabi.github.Repo;
+import com.jcabi.github.mock.MkGithub;
+import com.jcabi.xml.XMLDocument;
+import com.rultor.spi.Profile;
+import com.rultor.spi.Talk;
+import java.io.IOException;
+import org.apache.commons.lang3.StringUtils;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Test;
+import org.xembly.Directives;
+
+/**
+ * Tests for {@link ClosePullRequest}.
+ *
+ * @author Viktor Kuchyn (kuchin.victor@gmail.com)
+ * @version $Id$
+ * @since 2.0
+ * @checkstyle ClassDataAbstractionCoupling (200 lines)
+ */
+public final class ClosePullRequestTest {
+
+    /**
+     * ClosePullRequest can close pull request if rebase mode.
+     * @throws Exception If error
+     */
+    @Test
+    public void closesPullRequestForRebaseMode() throws Exception {
+        final Repo repo = new MkGithub().randomRepo();
+        final Issue issue = repo.issues().create("", "");
+        final Profile profile = new Profile.Fixed(
+            new XMLDocument(
+                StringUtils.join(
+                    "<p><entry key='merge'>",
+                    "<entry key='rebase'>true</entry>",
+                    "</entry></p>"
+                )
+            )
+        );
+        final Talk talk = ClosePullRequestTest.talk(repo, issue);
+        new ClosePullRequest(profile, repo.github()).execute(talk);
+        final Issue.Smart smart = new Issue.Smart(issue);
+        MatcherAssert.assertThat(
+            smart.state(), Matchers.is("closed")
+        );
+        MatcherAssert.assertThat(
+            new Comment.Smart(smart.comments().get(1)).body(),
+            Matchers.containsString(
+                "Closed manually because of rebase mode"
```

Comment:
@vkuchyn let's use better English here and actually give an explanation people outside of the people working on this understand :) I suggest:

> Rultor closed this pull request for you because your .rultor.yml specified the use of rebasing before merging. GitHub does not mark rebased pull requests as merged, because rebasing entails a change in commit hashes. Nevertheless all your files have been merged exactly as they would have been merged without the rebase option set.